### DISCUSSION TITLE
Be a Github Organization

### DISCUSSION THREAD

**Discussion by kachick** [mobu-of-the-world/mobu#53] [Ideas]
https://github.com/mobu

<img width="635" alt="スクリーンショット 2021-03-26 12 41 17" src="https://user-images.githubusercontent.com/1180335/112573917-94be3580-8e30-11eb-88a7-7426f20e8b1c.png">

The name of `mobu` is already taken 😢

**Comment by pankona** [comment]
😭

**Comment by pankona** [comment]
Today the mobu is moved under "mobu-of-the-world" organization.

**Comment by kachick** [comment]
🎉
